src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private final int index = new Random().nextInt(Color.values().length);
```

Comment:
You create ColorSupplier object only once, so yeah index will be random, but only once. You need to call nextInt() each time you call getrandomColor, to make it random